packages/server/modules/shared/authz.js

```diff
@@ -52,7 +52,7 @@ const validateStreamRole = ({ requiredRole }) =>
     requiredRole,
     rolesLookup: getRoles,
     iddqd: Roles.Stream.Owner,
-    roleGetter: (context) => context.stream.role
+    roleGetter: (context) => context.stream?.role
```

Comment:
why do we need this?

Reply:
I was getting errors cause of stream being undefined in some scenarios
i'm guessing in some test cases the context.stream resolution fails, and then because of that these auth checks fail as well (but in a messy way, throwing "undefined object" errors)